tests/test_permissions.py

```diff
@@ -0,0 +1,1254 @@
+    ),
+    field_id('Mutation', 'deleteOmicsRun'): TestData(
+        tenant_perm=MANAGE_OMICS_RUNS, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteRedshiftConnection'): TestData(
+        tenant_ignore=IgnoreReason.BACKPORT, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteRedshiftDataset'): TestData(
+        tenant_perm=MANAGE_REDSHIFT_DATASETS, resource_perm=DELETE_REDSHIFT_DATASET
+    ),
+    field_id('Mutation', 'deleteRedshiftDatasetTable'): TestData(
+        tenant_perm=MANAGE_REDSHIFT_DATASETS, resource_perm=DELETE_REDSHIFT_DATASET_TABLE
+    ),
+    field_id('Mutation', 'deleteSagemakerNotebook'): TestData(
+        tenant_perm=MANAGE_NOTEBOOKS, resource_perm=DELETE_NOTEBOOK
+    ),
+    field_id('Mutation', 'deleteSagemakerStudioUser'): TestData(
+        tenant_perm=MANAGE_SGMSTUDIO_USERS, resource_perm=DELETE_SGMSTUDIO_USER
+    ),
+    field_id('Mutation', 'deleteShareObject'): TestData(tenant_perm=MANAGE_SHARES, resource_perm=DELETE_SHARE_OBJECT),
+    field_id('Mutation', 'deleteTableDataFilter'): TestData(
+        tenant_perm=MANAGE_DATASETS, resource_perm=DELETE_TABLE_DATA_FILTER
+    ),
+    field_id('Mutation', 'deleteTerm'): TestData(
+        tenant_perm=MANAGE_GLOSSARIES, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteWorksheet'): TestData(tenant_perm=MANAGE_WORKSHEETS, resource_perm=DELETE_WORKSHEET),
+    field_id('Mutation', 'dismissTermAssociation'): TestData(
+        tenant_perm=MANAGE_GLOSSARIES, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'enableDataSubscriptions'): TestData(
+        tenant_perm=MANAGE_ENVIRONMENTS, resource_perm=ENABLE_ENVIRONMENT_SUBSCRIPTIONS
+    ),
+    field_id('Mutation', 'generateDatasetAccessToken'): TestData(
+        tenant_perm=MANAGE_DATASETS, resource_perm=CREDENTIALS_DATASET
+    ),
+    field_id('Mutation', 'importDashboard'): TestData(tenant_perm=MANAGE_DASHBOARDS, resource_perm=CREATE_DASHBOARD),
+    field_id('Mutation', 'importDataset'): TestData(tenant_perm=MANAGE_DATASETS, resource_perm=CREATE_DATASET),
+    field_id('Mutation', 'importRedshiftDataset'): TestData(
+        tenant_perm=MANAGE_REDSHIFT_DATASETS, resource_perm=IMPORT_REDSHIFT_DATASET
+    ),
+    field_id('Mutation', 'inviteGroupOnEnvironment'): TestData(
+        tenant_perm=MANAGE_ENVIRONMENTS, resource_perm=INVITE_ENVIRONMENT_GROUP
+    ),
+    field_id('Mutation', 'inviteGroupToOrganization'): TestData(
+        tenant_perm=MANAGE_ORGANIZATIONS, resource_perm=INVITE_ORGANIZATION_GROUP
+    ),
+    field_id('Mutation', 'markNotificationAsRead'): TestData(
+        tenant_ignore=IgnoreReason.APPSUPPORT, resource_ignore=IgnoreReason.APPSUPPORT
+    ),
+    field_id('Mutation', 'postFeedMessage'): TestData(
+        tenant_ignore=IgnoreReason.APPSUPPORT, resource_ignore=IgnoreReason.NOTREQUIRED
```

Comment:
postFeedMessage has permissions checks with the latest changes in `main` they are TARGET_TYPE_PERM